### DISCUSSION TITLE
How to change Footer links on blockscout

### DISCUSSION THREAD

**Discussion by King11919** [blockscout/blockscout#10386] [General]
![Screenshot_20240627_171827_Chrome](https://github.com/blockscout/blockscout/assets/69189392/283d2f40-05a0-4ec8-abb1-3f5b614a41a3)

**Comment by sl1depengwyn** [comment]
Refer to this https://github.com/blockscout/frontend/blob/main/docs/ENVS.md#footer

**Comment by King11919** [comment]
sir i used docker compose but i try lot but i can't even change footer-links

**Reply by King11919** [reply]
i need more help

**Reply by sl1depengwyn** [reply]
In order to change footer links you need to build your own docker build since these ENV values are build time, have you done this?

**Comment by King11919** [comment]
how to create docker build?

im current running this

https://github.com/blockscout/blockscout

**Reply by sl1depengwyn** [reply]
I'd suggest you to start from here https://docs.docker.com/guides/
Unfortunately, I have no capacity to provide your detailed guidance here, since it is not really relevant here

**Comment by King11919** [comment]
Sir can i do footer modifications on this (
https://github.com/blockscout/blockscout) or i need separate frontend?

**Reply by ekawibisono** [reply]
For recomendation separate frontend

**Comment by King11919** [comment]
Separate front end i want Upload to my github and deploy?

**Reply by ekawibisono** [reply]
Please clone frontend repository and please feel your modifications
https://github.com/blockscout/frontend/tree/main

**Comment by chrischarlesharrison** [comment]
How do you remove the blockscout links though
<img width="313" alt="Screenshot 2025-02-03 at 16 46 27" src="https://github.com/user-attachments/assets/28ce1c7c-246e-4bf0-a2be-38201130a55c" />
I'm able to create my own links, but they dont have the icons next to them, and i can't remove the blockscout links.

**Comment by King11919** [comment]
You Can Remove it

**Reply by chrischarlesharrison** [reply]
how then? do i just create a 'blockscout' entry in the footer env var with an empty array of links? Also how do i get those icons next to my own links?

**Comment by King11919** [comment]
No you need to Build own Container

**Comment by King11919** [comment]
if you want more info contact me on ***@***.***
